internal/backends/collection.go

```diff
@@ -48,6 +48,7 @@ type Collection interface {
 
 	Stats(context.Context, *CollectionStatsParams) (*CollectionStatsResult, error)
 	Compact(context.Context, *CompactParams) (*CompactResult, error)
+	Capped(context.Context) (bool, error)
```

Comment:
Oh, I remember this one was questionable when I was trying to offer it last time 😅 

`ListCollections` has info on wether a collection is capped. In `msg_collstats` there is an example how to use it - https://github.com/FerretDB/FerretDB/blob/main/internal/handlers/sqlite/msg_collstats.go#L88-L94.

While I agree that the method `Capped` would be really handy to have, I also understand that we don't want to extend the interface, so it's easier to support it.

(Unless something about it has changed, cc @AlekSi)

Reply:
#3601

Reply:
@noisersup please remove that method

Reply:
whoops my bad, fixed